<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="red_cube_big">
    <static>true</static>
    <link name="link">
      <visual name="visual">
        <geometry>
          <box>
            <size>4 4 1</size>
          </box>
        </geometry>
        <material>
          <ambient>1 0 0 0.5</ambient>
          <diffuse>1 0 0 0.5</diffuse>
          <specular>0.1 0.1 0.1 0.5</specular>
        </material>
      </visual>
    </link>
  </model>
</sdf>
